src/main/java/camp/nextstep/edu/nextstep8/lotto/LottoMatcher.java

```diff
@@ -0,0 +1,15 @@
+package camp.nextstep.edu.nextstep8.lotto;
+
+import java.util.stream.Collectors;
+
+public class LottoMatcher {
+    private LottoMatcher(){
+    }
```

Comment:
빈 중괄호는 한 줄로 붙여써도 됩니다. 앞으론 그렇게 해 주세요. 

```suggestion
    private LottoMatcher() {}